packages/react/src/drafts/Tooltip/Tooltip.examples.stories.tsx

```diff
@@ -16,7 +16,7 @@ import {
 import {default as VisuallyHidden} from '../../_VisuallyHidden'
 
 export default {
-  title: 'Drafts/Components/Tooltip/Examples',
+  title: 'Components/TooltipV2/Examples',
```

Comment:
Not blocking at all: Should we also move the component source out of drafts `src/TooltipV2`?

Reply:
Yeah, was just waiting to get an initial 👍🏻 in the direction - will move them over now.